### DISCUSSION TITLE
Where can I find TikTok's Gallery or Directory Download Files via PlayCover?

### DISCUSSION THREAD

**Discussion by spacegroupid** [PlayCover/PlayCover#381] [General]
I have successfully installed tiktok on mac m1 via PlayCover. For the needs of uploading videos, the tiktok application does not have video files on my mac.
including I have also downloaded the tiktok video, but did not find the file download folder.
where can the folder be found?

**Comment by Depal1** [comment]
At least for saving media, photos and videos are saved on your Photos library, the one from the Photos App. You have to enable the app’s access to it when prompted, or from the settings app under privacy and security.

**Reply by spacegroupid** [reply]
I looked for the Photo Library app but couldn't find it. Or where can I find the privacy and security settings?
